BrowserKit/Sources/Shared/Prefs.swift

```diff
@@ -182,8 +182,8 @@ public struct PrefsKeys {
     // Representing whether or not the last user session was private
     public static let LastSessionWasPrivate = "wasLastSessionPrivate"
 
-    // Only used to force nimbus features to true with tests
-    public static let NimbusFeatureTestsOverride = "NimbusFeatureTestsOverride"
+    // Only used in unit tests to override the user's setting for nimbus features
+    public static let NimbusUserEnabledFeatureTestsOverride = "NimbusUserEnabledFeatureTestsOverride"
```

Comment:
Renamed because previously using the `NimbusUserEnabledFeatureTestsOverride` always overrode the value to `true`. I wanted the value to override either `true` or `false` so we can test with both the user preference enabled or disabled.